hydra-redis/classes_objects.py

```diff
@@ -66,81 +66,98 @@ def objects_property(
             # set edge between the object and its parent object
             if endpoint_prop:
                 self.objects_property(
-                    object_node, endpoint_prop, entrypoint_node, api_doc)
+                    object_node, endpoint_prop, api_doc)
+
+    def load_from_server(
+            self,
+            class_endpoint_id,
```

Comment:
All you need to fetch data from server is `url` or the endpoint. Refactor the code and create simple functions